index.js

```diff
@@ -28,22 +28,24 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
+      var magenta = 'ffb3ff';
 
       console.log('='.repeat(consoleWidth()));
 
       if (results.length === 0) {
         console.log(chalk.red('No results were found, please try another phrase'));
       } else {
-        if (results.length > 3) {
-          trimRes = results.slice(0, 3);
+        if (results.length > resultsToDisplay) {
```

Comment:
Add some notes to your PR stating that you're laying the ground work for the results filtering